Tasks/JenkinsDownloadArtifacts/jenkins.handlebars.txt

```diff
@@ -0,0 +1,11 @@
+[
+{{#artifacts}}
+  {
+    "path" : "{{this.relativePath}}",
+    "itemType": "file",
+    "metadata" : {
+      "downloadUrl": "{{../endpoint.url}}/job/{{../definition}}/{{../version}}/artifact/{{this.relativePath}}"
```

Comment:
what do the ../ in the url signify?

Reply:
currently we are in artifacts context, we need to go up one level where variables are available. Just navigating the jobject.